libindy-crypto/src/cl/mod.rs

```diff
@@ -331,6 +331,7 @@ impl JsonEncodable for BlindedMasterSecret {}
 impl<'a> JsonDecodable<'a> for BlindedMasterSecret {}
 
 /// `Master Secret Blinding Data` uses by Prover for post processing of claims received from Issuer.
+/// TODO: Should be renamed `MasterSecretBlindingFactors`
```

Comment:
I don't see obvious advantage for "blinding factors" over "blinding data". Current name means "the data intended to blind master secret " What is your motivation?

Reply:
"blinding factor" is the term that used in literature and is well understood by cryptographers; `data` is just a blanket term.